src/components/Content/__tests__/__snapshots__/content.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Content renders correctly 1`] = `
 <div
-  className="StyledBox-sc-13pk1d4-0 gmaUvW"
+  className="StyledBox-sc-13pk1d4-0 dWFQNm"
```

Comment:
You can fix this from happening with `jest-styled-components`. Otherwise, you're updating your snapshots with every commit.

Reply:
can do

Reply:
added jest-styled-components and a setup file